src/index.js

```diff
@@ -0,0 +1,123 @@
+import './style.css';
+import updateStatus from './modules/status.js';
+import { addTask, deleteTask } from './modules/taskAdmin.js';
+
+let tasks = [];
```

Comment:
It would be better if you did not declare variables in global scope, it would be better to declare them locally inside functions because global variables can be overwritten by scripts.

Reply:
Thanks